src/objects/deck/demo/articles.twig

```diff
@@ -1,43 +1,63 @@
+{% set _deck_class = '' %}
```

Comment:
You'll want to use "hide whitespace" changes for this file — I wrapped the entire embed in a wrapper.